src/controllers/deleteEvents.js

```diff
@@ -0,0 +1,12 @@
+const Events = require('../database/event_schema');
+const dbConnect = require('../database/db_connect');
+
+exports.get = (req, res, next) => {
+
+const eventsId = req.params.id;
+
+Events.findByIdAndRemove({ _id: eventsId })
```

Comment:
Separate queries into a different directory, a controller has nothing to do with the database -directly-, but through the query, create a function that takes an `id` and removes from the database, and reuse it whenever you feel it.

Reply:
@NoureldeanSaed I have already solved this problem in the delete_events branch